node/resolvers/catalog/index.ts

```diff
@@ -150,6 +153,7 @@ const categoriesOnlyQuery = compose<
   string
 >(join('/'), map(prop('0')), filter(isTupleMap))
 
+// eslint-disable-next-line no-shadow
 const getAndParsePagetype = async (path: string, ctx: Context) => {
   const pagetype = await ctx.clients.catalog.pageType(path).catch(() => null)
```

Comment:
`path` is imported from ramda, so rename the parameter is better.

```suggestion
const getAndParsePagetype = async (pathQuery: string, ctx: Context) => {
  const pagetype = await ctx.clients.catalog
    .pageType(pathQuery)
    .catch(() => null)
```